cmd/goexports/goexports.go

```diff
@@ -53,6 +57,7 @@ import (
 
 func init() {
 	Symbols["{{.PkgName}}"] = map[string]reflect.Value{
+		{{- if len .Val | ne 0}}
```

Comment:
This check seems not needed because the range syntax already support empty map.

Reply:
yes, that was the goal.